README.md

```diff
@@ -29,6 +29,9 @@ Once the container is running, you can start a bash session inside
 the container with `docker exec -it cacophony-api bash`.  You can then use
 psql to query the database `sudo -i -u postgres psql cacophonytest`
 
+Note that the source code is mounted on to the docker image, so any changes 
+causes the server to reload.
```

Comment:
Thanks for adding this. Can you please extend this further to mention `./run --isolate` which runs the server independently of the source tree (including `node_modules`)?